### DISCUSSION TITLE
Freeze Table Column or Row

### DISCUSSION THREAD

**Discussion by frankyso** [filamentphp/filament#5048] [Ideas]
Can we make the filament table column or row freezing while scrolling though the data?

**Comment by sergioraldana** [comment]
did you find an answer?

**Reply by danharrin** [reply]
CSS positioning?

**Reply by R70YNS** [reply]
```
<style>
    table {
        position: relative;
    }

    table th {
        position: sticky;
        top: 0;
        z-index: 2;
        background: #fafafa;
    }

    table th:nth-child(1) {
        left: 0;
        z-index: 3;
    }

    table tbody tr td:nth-child(1) {
        position: sticky;
        left: 0;
        z-index: 1;
        background: white;
        background-clip: padding-box;
    }
</style>
```
Not the most elegant solution in terms of integration, but above CSS works fine. Would need modification for dynamic background colours. 

Thanks to Milosz for original answer on:  https://stackoverflow.com/questions/45071085/freeze-first-row-and-first-column-of-table
